support-frontend/assets/helpers/billingPeriods.js

```diff
@@ -3,8 +3,10 @@
 const Annual: 'Annual' = 'Annual';
 const Monthly: 'Monthly' = 'Monthly';
 const Quarterly: 'Quarterly' = 'Quarterly';
+const SixForSix: 'SixForSix' = 'SixForSix';
```

Comment:
`SixForSix` is actually called `SixWeekly`, this is to match with existing code elsewhere

Reply:
Ah - I left both in there. Is it changed across the board now?

Reply:
Yes it should be